mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -64,9 +69,15 @@ def __remote_init(self):
         # Query for available resource
         # Automatic selection and allocation of a resource
         try:
-            self.selected_resource = self.client.allocate({
-                "platform_name": self.platform_name
-            })
+            if self.tags:
```

Comment:
if there is somethibg like `self.allocate_requirements` dict then we doesnt need these if-else statements... would be bit more simple and easier to further develop requirement parameters..